### DISCUSSION TITLE
Read Me File cannot be changed :<

### DISCUSSION THREAD

**Discussion by Dwieght** [community/community#70739] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

The readme is in Chinese. There is a link to an English version at the top of the document. It is linked to the file "EN_Readme.md" in the "translations" folder. This link doesn't work.

This problem only occurs if you click on the link directly in the repository's start page (in the readme preview below).

**Comment by DwieghtFuentes** [comment, marked as answer]
try chatgpt
